test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1494,67 +1494,69 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
-            # macOS does not support multiple SslProtocols
-            if (-not $IsMacOS) 
-            {
-                @{SslProtocol = 'Tls, Tls11, Tls12'; ActualProtocol = 'Tls12'}
-                @{SslProtocol = 'Tls11, Tls12'; ActualProtocol = 'Tls12'}
-                @{SslProtocol = 'Tls, Tls11, Tls12'; ActualProtocol = 'Tls11'}
-                @{SslProtocol = 'Tls11, Tls12'; ActualProtocol = 'Tls11'}
-                @{SslProtocol = 'Tls, Tls11'; ActualProtocol = 'Tls11'}
-                @{SslProtocol = 'Tls, Tls11, Tls12'; ActualProtocol = 'Tls'}
-                @{SslProtocol = 'Tls, Tls11'; ActualProtocol = 'Tls'}
-                @{SslProtocol = 'Tls, Tls12'; ActualProtocol = 'Tls'}
-            }
-            # macOS does not support multiple SslProtocols and possible CoreFX for this combo on Linux
-            if($IsWindows)
-            {
-                
-                @{SslProtocol = 'Tls, Tls12'; ActualProtocol = 'Tls12'}
-            }
-        ) {
-            param($SslProtocol, $ActualProtocol)
-            $params = @{
-                Uri = Get-WebListenerUrl -Test 'Get' -Https -SslProtocol $ActualProtocol
-                SslProtocol = $SslProtocol
-                SkipCertificateCheck = $true
-            }
-            $response = Invoke-WebRequest @params
-            $result = $Response.Content | ConvertFrom-Json
+        BeforeAll {
+            ## Test cases for the 1st 'It'
+            $testCases1 = @(
+                @{ Test = @{SslProtocol = 'Default'; ActualProtocol = 'Default'}; Pending = $false }
+                @{ Test = @{SslProtocol = 'Tls'; ActualProtocol = 'Tls'}; Pending = $false }
+                # these two currently fail on Travis CI macOS build
+                @{ Test = @{SslProtocol = 'Tls11'; ActualProtocol = 'Tls11'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls12'; ActualProtocol = 'Tls12'}; Pending = $IsMacOS }
+                # macOS does not support multiple SslProtocols
+                @{ Test = @{SslProtocol = 'Tls, Tls11, Tls12'; ActualProtocol = 'Tls12'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls11, Tls12'; ActualProtocol = 'Tls12'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls, Tls11, Tls12'; ActualProtocol = 'Tls11'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls11, Tls12'; ActualProtocol = 'Tls11'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls, Tls11'; ActualProtocol = 'Tls11'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls, Tls11, Tls12'; ActualProtocol = 'Tls'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls, Tls11'; ActualProtocol = 'Tls'}; Pending = $IsMacOS }
+                @{ Test = @{SslProtocol = 'Tls, Tls12'; ActualProtocol = 'Tls'}; Pending = $IsMacOS }
+                # macOS does not support multiple SslProtocols and possible CoreFX for this combo on Linux
```

Comment:
Can you change this to `possible CoreFX issue`, please? My own missing word it bugging me 😆

Reply:
Fixed. Thanks for spotting it!